membership-attribute-service/test/services/SupporterRatePlanToAttributesMapperTest.scala

```diff
@@ -313,43 +313,6 @@ class SupporterRatePlanToAttributesMapperTest extends Specification {
       }.toList
     }
 
-    "Always choose the most valuable membership" in {
```

Comment:
probably not an important thing to get right, I'd be surprisied if anyone has multiple memberships anyway